src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts

```diff
@@ -62,7 +62,7 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
   constructor(private readonly hearingStore: Store<fromHearingStore.State>, private readonly hearingsService: HearingsService, private readonly route: ActivatedRoute) {
     this.hearingRoles = this.route.snapshot.data.hearingRole;
     this.hearingTypes = this.route.snapshot.data.hearingTypes;
-    this.partyChannels = this.route.snapshot.data.partyChannel;
+    this.partyChannels = [...this.route.snapshot.data.partyChannels, ...this.route.snapshot.data.partyChannels];
```

Comment:
Shouldn't it be PartySubChannels?

Reply:
Thank you @MateiHMCTS, I did the changes now